app/views/blocked/users/index.html.erb

```diff
@@ -17,13 +17,19 @@
 </div>
 
 <% # form for blocking users %>
-<div class="actions">
-  <%= form_tag confirm_block_user_blocked_users_path(@user), method: :get do %>
-    <%= text_field_tag :blocked_id, "", autocomplete_options("pseud", size: 60, data: { autocomplete_token_limit: 1 }) %>
```

Comment:
Discussed elsewhere: we'll remove the ["size" HTML attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/size) because we don't really need it for presentation (it's on the hidden input elements and jquery.tokeninput.js doesn't copy it to the displayed input elements, plus CSS takes precedence) or validation (that needs to happen server-side regardless).